web/api/v1/api.go

```diff
@@ -148,9 +149,16 @@ func (api *API) Register(r *route.Router) {
 	r.Get("/alertmanagers", instr("alertmanagers", api.alertmanagers))
 }
 
+type queryStats struct {
+	ExecQueueTime time.Duration `json:"execQueueTimeNs"`
```

Comment:
Seconds even for internal measurements? I guess if the timers are in low nanos, the numbers won't matter much, because that's fast. However, as an API consumer I do care about times in the millies. And I would like the stats timers to be of the same unit. Currently the API response would expose the `time.duration` as measured by Go, which happens to be in `ns`.

- [ ] Keep nanos for precision, or
- [ ] Use millies for API consumer relevance, or
- [ ] Use seconds, with decimals (how many?) for consistency